### PR TITLE
Added steps to migrate to new DB device (bsc#1202048)

### PR DESCRIPTION
**Pull request by tbazant**
fixes https://bugzilla.suse.com/show_bug.cgi?id=1202048

### CODE REVIEW

**Review by lezare — CHANGES_REQUESTED** [review]
There is a small typo, `reason` should be `reasons` on line 1222, and the command sequence should be as per the following example:

1. `cephadm unit --name osd.${id} stop` on the relevant OSD node.
2. `cephadm shell --name osd.${id}` to enter into the stopped OSD container.
3. `ceph-volume lvm migrate --osd-id ${id} --osd-fsid ${fsid} --from db --target ceph-6ff5feb5-b489-4640-99c3-3551f881a307/osd-db-da349474-b33f-4efc-b4fc-a9c99a1d4e24`
4. `exit` the cephadm shell when done with the commands
5. `cephadm unit --name osd.${id} start` on the relevant OSD node to start the OSD again.

Should we maybe also add information on how to properly create a LV for the new DB? Something like the following command can be used to create the new LV with the desired size:

`lvcreate -n osd-db-$(cat /proc/sys/kernel/random/uuid) ceph-6ff5feb5-b489-4640-99c3-3551f881a307 --size <db_size>`

I am not sure if we should mention something (add a note?) about the ceph-volume naming convention, as this changed some over time (in case this is confusing for some customers). Some clusters may for example have VG / LV names as follows which differs from the current naming convention:

`ceph-block-dbs-<uuid>/osd-block-db-<uuid>` (old)
`ceph-<uuid>/osd-db-<uuid>` (current)

From a functional perspective, which ever naming convention is used for the VGs / LVs should in the end however not matter.

File: xml/admin_saltcluster.xml
Comment (trociny) [inline xml/admin_saltcluster.xml:1293]: @lezare  Don't we need `ceph-volume lvm new-db` before `lvm migrate`? I did not test these instructions but AFAIR when I did migration I needed to run `new-db`.

[1] https://docs.ceph.com/en/quincy/ceph-volume/lvm/newdb/

Comment (trociny) [reply xml/admin_saltcluster.xml:1293]: Or may be it was a little different case of migrating from osd withoud dedicated db...

Anyway I am fine with changes if they are tested and proved to work. Otherwise I will need to test it before approval.

File: xml/admin_saltcluster.xml
Comment (dariavladykina) [inline xml/admin_saltcluster.xml:1223]:
```suggestion
    new one&mdash;for example, when OSDs have different DB sizes and you need to 
    align them.
```

**Review by dariavladykina — COMMENTED** [review]
LGTM!

**Pull request merged**
